lib/cretonne/meta/cdsl/registers.py

```diff
@@ -195,24 +195,29 @@ class RegClass(object):
     :param start: The first unit to allocate, relative to `bank.first.unit`.
     """
 
-    def __init__(self, bank, count=None, width=1, start=0):
-        # type: (RegBank, int, int, int) -> None
+    def __init__(self, bank, count=0, width=1, start=0, bitmask=None):
+        # type: (RegBank, int, int, int, int) -> None
```

Comment:
I'm not very familiar with the mypy type system; is it valid to have a value of `None` if the mypy type is `int`?

Reply:
Good point. That should be `Optional[int]`.